src/jailer/src/env.rs

```diff
@@ -344,8 +344,7 @@ impl Env {
 
     fn exec_into_new_pid_ns(&mut self, chroot_exec_file: PathBuf) -> Result<(), JailerError> {
         // Compute jailer's total CPU time up to the current time.
-        self.jailer_cpu_time_us =
-            utils::time::get_time_us(utils::time::ClockType::ProcessCpu) - self.start_time_cpu_us;
+        self.jailer_cpu_time_us += utils::time::get_time_us(utils::time::ClockType::ProcessCpu);
```

Comment:
There is a change of behaviour when --daemonize option is not passed but --new-pid-ns is passed, so I think we need to use start_time_cpu_us here and not in the daemonize condition:
```suggestion
        self.jailer_cpu_time_us += utils::time::get_time_us(utils::time::ClockType::ProcessCpu);
        self.jailer_cpu_time_us += utils::time::get_time_us(utils::time::ClockType::ProcessCpu) - self.start_time_cpu_us;
```

Reply:
Good catch! Initially I thought just removing that substraction altogether.

So it gets a bit confusing depending on the combination of options. I think we need to take the accounting out of the `--new-pid-ns` option, and add the time before a fork/clone.

Reply:
Fixed (hopefully) in latest PR.